crosswalks/publiccode.csv

```diff
@@ -0,0 +1,74 @@
+Property,publiccode
+codeRepository,url
+programmingLanguage,
+review,
+runtimePlatform,platforms
+targetProduct,platforms
+applicationCategory,categories / description.[lang].genericName
+applicationSubCategory,categories
+downloadUrl,landingUrl
+fileSize,
+installUrl,landingUrl
+memoryRequirements,
+operatingSystem,platforms
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,dependsOn.open / dependsOn.proprietary / dependsOn.hardware
+softwareVersion,softwareVersion
+storageRequirements,
+supportingData,
+author,
+citation,
+contributor,
+copyrightHolder,legal.mainCopyrightOwner
+copyrightYear,
+dateCreated,
+dateModified,releaseDate
+datePublished,releaseDate
+editor,
+encoding,
+fileFormat,
+funder,
+keywords,description.[lang].features
+license,legal.license
+producer,
+provider,
+publisher,
+sponsor,
+version,softwareVersion
+isAccessibleForFree,
+isPartOf,applicationSuite
+hasPart,
+position,
+description,description.[lang].shortDescription / description.[lang].longDescription
+identifier,
+name,name / description.[lang].localisedName
+sameAs,
+url,landingUrl
+relatedLink,
+givenName,
+familyName,
+email,
+affiliation,
+identifier,
+name,
+address,
+reviewAspect,
+reviewBody,
+endDate,
+roleName,
+startDate,
+softwareSuggestions,
+maintainer,maintenance.contractors / maintenance.contacts
+continuousIntegration,
+buildInstructions,
+developmentStatus,developmentStatus
+embargoEndDate,
+funding,
+issueTracker,
+referencePublication,
+readme,
+hasSourceCode,
+isSourceCodeOf,isBasedOn
```

Comment:
isSourceCodeOf is to describe relationships between an application and its code. https://schema.org/isBasedOn is what you are looking for, but it's not part of Codemeta.